ReturnBookControl.java

```diff
@@ -1,76 +1,76 @@
 public class ReturnBookControl {
 
-	private ReturnBookUI Ui;
-	private enum CONTROL_STATE { INITIALISED, READY, INSPECTING };
-	private CONTROL_STATE sTaTe;
+	private ReturnBookUI returnBookUi;
+	private enum RetunrBookControlState { INITIALISED, READY, INSPECTING };
+	private RetunrBookControlState state;
 	
-	private library lIbRaRy;
-	private loan CurrENT_loan;
+	private library library;
+	private loan currentLoan;
 	
 
 	public ReturnBookControl() {
-		this.lIbRaRy = lIbRaRy.INSTANCE();
-		sTaTe = CONTROL_STATE.INITIALISED;
+		this.library = library.instanceLibrary();
+		state = RetunrBookControlState.INITIALISED;
 	}
 	
 	
-	public void Set_UI(ReturnBookUI ui) {
-		if (!sTaTe.equals(CONTROL_STATE.INITIALISED)) {
+	public void returnBookSetUi(ReturnBookUI ui) {

```

Comment:
@GoJoeyGo I think setReturnBookUi seems little good then returnBookSetUi. can you fix it. Thanks

Reply:
ReturnBookUi will be changed to setReturnBookUi

Reply:
Great